_CONTRIBUTING.md

```diff
@@ -25,4 +34,46 @@ git push origin your_feature_branch
 # open repo in your browser and you should see a Create PR option.
 ```
 
-## More guidelines will come here...
+# Conventions
+
+Database:
+
+- Use migrations and factories for database structure and population.
+- Avoid using Database triggers; opt for Laravel's observers.
+- Use database constraints as much as possible within reasonable limits. This usually includes foreign and composite keys.
+- We prefer a separate table for enum-like values over enum type fields for easier value modification.
+
+Laravel Features:
+
+- Use policies instead of gates.
+- Use observers instead of events or triggers.
+- Use attributes for custom getters/setters.
+
+Structure:
+
+- Utilize resources whenever possible, following the index-create-store-show-edit-update-destroy naming convention if applicable.
+- Refrain from adding logic to the User model; delegate logic to other models where possible.
+
+Translation:
```

Comment:
We should probably specify that translations are required in English and Hungarian. Others don't work AFAIK.

Reply:
They should work, coming from the language contributions in the database. It's another thing that we don't have much contribution.

Reply:
Oh, I thought there was a technical problem but now I can see that it works